pkg/auth/handler/webapp/enter_oob_otp.go

```diff
@@ -82,6 +83,17 @@ func (h *EnterOOBOTPHandler) GetData(r *http.Request, rw http.ResponseWriter, se
 			viewModel.OOBOTPTarget = phone.Mask(target)
 		}
 
+		if channel == model.AuthenticatorOOBChannelSMS {
+			bucket := h.AntiSpamSMSBucket.MakeBucket(target)
+			pass, _, err := h.RateLimiter.CheckToken(bucket)
+			if err != nil {
+				return nil, err
+			}
+			if !pass && baseViewModel.Error == nil {
+				baseViewModel.SetError(bucket.BucketError())
+			}
+		}
```

Comment:
I thought the end-user should not be in this page if the limit is exceeded. Instead, they should see the error in the previous page in the error message bar. We should not need to inject the bucket maker and query the bucket limit.